hapi-proto/src/main/proto/ResponseCode.proto

```diff
@@ -203,9 +203,15 @@ enum ResponseCodeEnum {
   INVALID_WIPE_KEY = 189;
   INVALID_FREEZE_KEY = 190;
   INVALID_SUPPLY_KEY = 191;
-  TOKEN_NAME_ALREADY_IN_USE = 192; // Token Name is already used
```

Comment:
ResponseCode comments should be left because they are used as documentation.
You can add some information about the newly added Codes by you and update the `HAPI.html`